gym/envs/mujoco/mujoco_env.py

```diff
@@ -112,6 +112,13 @@ def render(self, mode='human', width=DEFAULT_SIZE, height=DEFAULT_SIZE):
             data = self._get_viewer(mode).read_pixels(width, height, depth=False)
             # original image is upside-down, so flip it
             return data[::-1, :, :]
+        elif mode == 'depth_array':
+            self._get_viewer(mode).render(width, height)
+            # window size used for old mujoco-py:
+            # Extract depth part of the read_pixels() tuple
+            data = self._get_viewer(mode).read_pixels(width, height, depth=True)[1]
+            # original image is upside-down, so flip it
+            return data[::-1, :]
```

Comment:
actually, you lost me now... Why is depth data 2D? Do we still need to flip it upside down?

Reply:
RGB is [h, w, 3] where 3 is for the r, g, b.
Depth I initially assumed is [h, w, 1] , after local testing, it turned out to be [h, w].

Reply:
oh got it. Ok.